Authentication Added

mo-kalaleeb opened the pull request.

mo-kalaleeb committed:
Authentication Added

gh-assan commented on LICENSE:
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 yes-soft-de
+Copyright (c) 2019 MickSawy3r
```
yes soft

gh-assan commented on java/com/sixbits/yessoft/MainActivity.java:
```diff
@@ -0,0 +1,82 @@
+package com.sixbits.yessoft;
```
please use de.yessoft.android 
as root package name, for all classes

gh-assan commented on java/com/sixbits/yessoft/ui/NewsFeedFragment/NewsRecyclerView.java:
```diff
@@ -0,0 +1,77 @@
+        return 0;
+    }
+
+
```
please do not keep this code, if you want you can find them in git history

gh-assan commented:
good, thank you

gh-assan commented:
in general use descriptive branch name and commit messages

mo-kalaleeb committed:
Firebase Firestore Added.

Final Sprint Version will be ready ASAP containing Nitification and NewsFeed As Soon it will be ready, that should be in the margins of 24 hours

mo-kalaleeb committed:
Change log update

mo-kalaleeb committed:
Notification Added, Appbar Added, master ready version

mo-kalaleeb committed:
change log recorded

Kenan-Hussein reviewed (approved):
good work, thank you
